pytest_easyread.py

```diff
@@ -180,6 +180,9 @@ def summary_failures(self):
                     self._ljust_sep(rep, sepchar, title, **markup)
                     self.is_first_failure = False
                     index += 1
+                    failed_test_path = self._locationline(rep.nodeid, *rep.location)
+                    self._tw.write(" "*3 + "Path: " + failed_test_path)
+                    self._tw.line()
```

Comment:
Shall we add the new functionality to our comment above (better yet, we could make it a docstring for this method)?

Reply:
that sound great, thanks for the suggestion! 👍

Reply:
I actually refactored this to separate function now as too much was happening in one place. I'd love it if we could sit for 15 minutes next week and do one of those top-of-the-function docstrings together. I read quite a bit about them but I still do not feel sure about how to write my own. Only if you are up for it of course :).

Reply:
Happy to! 👍 for refactoring too - I'll have another look next week.